client/src/js/app.js

```diff
@@ -16,7 +18,7 @@ class App {
     this.main = document.querySelector(settings.gridSection)
     this.nav = document.querySelector(settings.navSection)
     this.ul = document.querySelector(settings.ulSelector)
-    this.channelUILists = document.querySelector(settings.channelUIListsSelector)
+    this.channelUILists = document.querySelector(settings.channelUIListsSelector)    
```

Comment:
Lets setup a linter to avoid such changes. I don`t think you need that spaces at the end of this line